### PR TITLE
OpenDingux: GKD350h support

### PR DESCRIPTION
**Pull request by glebm**
gkd-350h-devilutionx

https://github.com/glebm/devilutionX/releases/tag/0.5.0.3508

Start and Select are swapped in this version because Start + D-Pad controls backlight on the GKD350h.

There is nothing to map mouse emulation to in this version because of this but it isn't necessary to play the game.

This version looks uglier than on RG350 because it uses software scaling beacuse the IPU on the GKD350h doesn't work.

### CODE REVIEW

**Comment by AJenbo** [comment]
Please rebase now that #534 is merged :)

**Comment by glebm** [comment]
Done!

**Comment by glebm** [comment]
Also update all readmes/manuals for controller stuff to reflect the recent changes

File: SourceX/miniwin/misc.cpp [hunk 1]
```diff
@@ -19,6 +19,12 @@
 #ifndef SDL1_VIDEO_MODE_FLAGS
 #define SDL1_VIDEO_MODE_FLAGS SDL_SWSURFACE
 #endif
+#ifndef SDL1_VIDEO_MODE_WIDTH
+#define SDL1_VIDEO_MODE_WIDTH nWidth
```
Comment (AJenbo) [inline SourceX/miniwin/misc.cpp]: Maybe a bit of a hack considering the default can't be used in places where `nWidth` isn't defined (a local of `SpawnWindow`). Maybe instead we should `#ifdef`

Comment (glebm) [reply SourceX/miniwin/misc.cpp]: I agree but it's used in 2 places so ifdefs would get a bit verbose.
I think it's fine as this is confined to this `cpp` file.

**Pull request merged**